src/app/strategies/[strategyId]/page.tsx

```diff
@@ -1,45 +1,164 @@
+/** @jsxImportSource @emotion/react */
 "use client";
 
 import React, { useState, useEffect } from 'react';
 import Image from 'next/image';
 import axios from 'axios';
 import { useParams } from 'next/navigation';
 import no_record from '../../../assets/images/no_record.png';
-import '../../styles/orders.css';
+import { css } from '@emotion/react';
+import Pagination from '../../../component/pagination';
+import '../../globals.scss';
 import util from '../../../utils/util';
+// import Highcharts from 'highcharts';
+import Chart from '@/src/component/chart';
+
+interface DetailProps {
+  _id: string;
+  createdAt: string;
+  exchange: string;
+  symbol: string;
+  quote: number;
+  quote_total: number;
+  price_native: string;
+  price_total: string;
+  base: string;
+  base_limit: string;
+  base_total: number;
+  profit: number;
+  profit_percentage: number;
+  stop_profit_percentage: number;
+  price_usd: number;
+  status: string;
+}
+
+interface OrderProps{
+  _id: string;
+  createdAt: string;
+  symbol: string;
+  price: string;
+  amount: number;
+  funds: string;
+  record_amount: number;
+  avg_price: number;
+  base_total: number;
+  value_total: number;
+  cost: number;
+}
 
 const strategyDetails: React.FC = () => {
   const { strategyId } = useParams();
-  const [ orders, setOrders ] = useState([]);
-  const [ details, setDetails ] = useState([]);
+  const [ orders, setOrders ] = useState<OrderProps[]>([]);
+  const [ details, setDetails ] = useState<DetailProps|undefined>(undefined);
+  const [ fontColor, setFontColor] = useState<string>('');
+
+  // Props for chart
+  const [series1, setSeries1] = useState<number[]>([]);
+  const [series2, setSeries2] = useState<number[]>([]);
+  const [series3, setSeries3] = useState<number[]>([]);
+  const [categories, setCategories] = useState<string[]>([]);
+  const [max, setMax] = useState(0);
+  const [min, setMin] = useState(0);
 
   useEffect(() => {
     const queryDetails = async () => {
         try {
             // todo: use baseURL instead of hardcode
             const detailsResponse = await axios.get(`http://127.0.0.1:3001/api/v1/strategies/${strategyId}`);
-            setDetails(detailsResponse.data.data.info);
-        } catch (error) {
+            const data = detailsResponse.data.data;
+
+            // Set variables
+            if (data) {
+              const info = data.info;
+              // Todo: complete symbol exchange model in back-end
+              // const symbol_price = data.symbol_price;
+              info.price_usd = 10; // Todo: update later
```

Comment:
We usually put TODO before the code and there is a space between it and the content, such as:
// TODO something